docs/virtual_machines/instancetypes.md

```diff
@@ -0,0 +1,298 @@
+# Instancetypes and preferences
+
+**FEATURE STATE:** 
+
+* `v1alpha1` (Experimental) as of the [`v0.56.0`](https://github.com/kubevirt/kubevirt/releases/tag/v0.56.0) release
+
+## Introduction
+
+KubeVirt's [`VirtualMachine`](https://kubevirt.io/api-reference/main/definitions.html#_v1_virtualmachine) API contains many advanced options for tuning the performance of a VM that goes beyond what typical users need to be aware of. Users have previously been unable to simply define the storage/network they want assigned to their VM and then declare in broad terms what quality of resources and kind of performance characteristics they need for their VM.
```

Comment:
> Thanks @lyarwood , this section will help users and developers to get started with instancetype and preferences for sure. The only thing which could be improved from my point of view, is the example. I like the approach to keep the example in sync with https://github.com/kubevirt/kubevirt/tree/main/examples , but I wish the example would be more useful and ready to be copy and pasted. Would it be possible to have a fedora VM (because using a containerdisk from https://quay.io/repository/containerdisks/fedora would enable the user to run the example), and use the preferences to enable uefi (virtio is the defauly anyway).

The examples can be copy pasted as is, I can provide the initial `cat << EOF | kubectl apply -f -` if required but a user could just as easily copy this into a single file and apply that instead. I'll switch the example to containerdisks but I think it should still default to CirrOS.

Reply:
Thanks! I like to have all images for users in https://github.com/kubevirt/containerdisks (or at a source controlled by the vendor, e.g. fedora, directly), so we should add cirros there, if we want to use it in examples.